recipes/MEAutility/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "MEAutility" %}
+{% set version = "1.4.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cd1f723992aa7366f1cbbf8f2843e1f61d8123b459435653860a3b544bc752f"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
```

Comment:
I think you only need `pip` and `python` under `host` since the others aren't needed to build the package.